code/computational_geometry/src/parabola_drawing/Parabola.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <graphics.h>
+#include <math.h>
+
+using namespace std;
+
+void putPixel(int x, int y)
+{
+    putpixel((x + 300), (240 - y), 15);
+    putpixel((x + 300), (240 + y), 15);
+}
+
+void para(int cx, int cy, double a)
+{
+    // Centre has been shifted to (300, 240)

```

Comment:
Same changes needed as in #4087

Reply:
I have made the required changes. Kindly review my changes